frameworks.rst

```diff
@@ -104,6 +101,11 @@ Deprecated Systems
 These systems still exist but got replaced by others or are
 unmaintained.
 
+`notmm <https://pypi.org/project/django-hotsauce/0.5.5/>`_
```

Comment:
Maybe instead of moving notmm to deprecated and changing the URL, just rename the entry to "django-hotsauce" and add that it was formerly called notmm? And the link should probably go to https://pypi.org/project/django-hotsauce/

Reply:
Yes, brought back to it's place, renamed and updated link. Please check and let me know if I got you correctly.
Thanks 🙂

Reply:
Great idea! I'm making the changes you just mentioned.